onnx/defs/tensor/defs.cc

```diff
@@ -1363,9 +1381,17 @@ Example 2:
 
 ONNX_OPERATOR_SET_SCHEMA(
     ScatterND,
-    13,
+    16,
     OpSchema()
-        .SetDoc(ScatterND_ver13_doc)
+        .SetDoc(ScatterND_ver16_doc)
+        .Attr(
+            "reduction",
+            "Type of reduction to apply: none (default), add, multiply. "
```

Comment:
Nit: replacing "multiply" by "mul" will make it a bit more consistent with the names used for the corresponding ONNX ops.